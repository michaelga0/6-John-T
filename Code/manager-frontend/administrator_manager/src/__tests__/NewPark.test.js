import React from 'react';
import NewPark from '../NewPark/NewPark';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow } from 'enzyme';
import toJSON from "enzyme-to-json"
import { BrowserRouter } from 'react-router-dom';


Enzyme.configure({adapter: new Adapter()});

it('renders correctly', () => {
    const tree = shallow(<BrowserRouter> <NewPark/> </BrowserRouter>)
expect(toJSON(tree)).toMatchSnapshot(); 
});