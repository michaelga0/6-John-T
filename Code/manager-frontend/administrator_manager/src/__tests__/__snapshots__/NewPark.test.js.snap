// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "createHref": [Function],
      "createURL": [Function],
      "encodeLocation": [Function],
      "go": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
   
  <NewPark />
   
</Router>
`;
